fuzzing/tools/launcher.py

```diff
@@ -0,0 +1,26 @@
+# Lint as: python3
+"""
+This is the launcher script to provide a uniform command line interface 
+behind a number of arbitrary fuzzing engines.
+"""
+
+from absl import app
+from absl import flags
+import os
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_integer('timeout', 20, 'test timeout.', lower_bound=0)
+
+
+def main(argv):
+    if len(argv) != 2:
+        raise app.UsageError(
+            "This script receives 2 arguments. It should look like:" +
```

Comment:
I believe this is incorrect: The script receives one argument only, the other one being argv[0], which is the executable path.